templates/common_macros.html.j2

```diff
@@ -100,7 +100,7 @@
               <td style="word-break:normal">{{ attr }}</td>
               <td>
                 {%- if value.as_svg -%}
-                  {{ value.as_svg | safe }}
+                  {{ value }}
```

Comment:
@Wuestengecko this breaks a template.
Try

```shell
CME_MODEL=git+https://github.com/DSD-DBS/Capella-IFE-sample.git
```
and then go to:

System Interfaces. There is just one named `I_IFE_Crew`.

Reply:
Question is, how we get the "Click to enlarge" feature, when we cannot apply above modification.

Reply:
An update of capellambse (see https://github.com/DSD-DBS/py-capellambse/pull/510) will fix this.